code/WorkInProgress/warcrimes.dm

```diff
@@ -811,13 +811,8 @@ Urs' Hauntdog critter
 	name_the_meat = 0
 
 	specific_emotes(var/act, var/param = null, var/voluntary = 0)
-		switch (act)
-			if ("scream")
-				if (src.emote_check(voluntary, 50))
-					var/turf/T = get_turf(src)
-					return "<span style='color:#605b59'><b>[src]</b> screeeams!</span>"
-					var/hogg = pick("sound/voice/hagg_vorbis.ogg","sound/voice/hogg_vorbis.ogg","sound/voice/hogg_vorbis_the.ogg","sound/voice/hogg_vorbis_screams.ogg","sound/voice/hogg_with_scream.ogg","sound/voice/hoooagh2.ogg","sound/voice/hoooagh.ogg",)
```

Comment:
the pick and playsound looks like it was supposed to run before the return

Reply:
fixed